web/src/pages/apd/export/ApdExport.js

```diff
@@ -34,7 +40,7 @@ const ExportAndSubmit = ({
       <Waypoint /> {/* Waypoint w/o id indicates top of page */}
       <AlertMissingFFY />
       <Section resource="exportAndSubmit">
-        {process.env.TEALIUM_ENV !== 'prod' && (
+        {process.env.TEALIUM_ENV !== 'prod' && adminCheckFlag === true && (
```

Comment:
this should be using the flag not the tealium environment now

Reply:
whoops, good catch